src/styles/blocks/calendar.scss

```diff
@@ -1,47 +1,45 @@
 .calendar {
   display: flex;
+  margin: 0 auto;
+  width: 7 * $square-size + 6 * $square-gap + 2 * $square-paddings;
```

Comment:
Number 7 can be set as a variable. 6 can be calculated from it.

Reply:
and this